packages/devtools_app/test/logging_screen_test.dart

```diff
@@ -312,6 +312,33 @@ void main() {
         });
       });
     });
+
+    group('MessageColumn', () {
```

Comment:
maybe add a simple test for the secondarySortColumn functionality?

Also add a test with the following data
```
     final c = LogData('flutter.frame', '#10 3.6ms', 1);
     final d = LogData('flutter.frame', '#10foo 3.6ms', 1);
     final e = LogData('flutter.frame', '#9  3.6ms ', 1);
     final f = LogData('flutter.frame', '#9foo  3.6ms ', 1);
     to see what happens where there is data that is almost a id but isn't.